src/SmartComponents/PatchSetWizard/steps/ReviewPatchSet.js

```diff
@@ -11,6 +11,19 @@ import {
     TextListItemVariants
 } from '@patternfly/react-core';
 import useFormApi from '@data-driven-forms/react-form-renderer/use-form-api';
+import { intl } from '../../../Utilities/IntlProvider';
+import messages from '../../../Messages';
+
+const renderTextListItem = (label, text) => (
+    <Fragment>
```

Comment:
```suggestion
    <React.Fragment>
```
because the rest of the app uses this notation :) better to choose one and be consistent here..